StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -11,10 +11,15 @@ import UIKit
 
 protocol NativeFlowControllerDelegate: AnyObject {
 
-    func authFlow(
-        controller: NativeFlowController,
+    func nativeFlowController(
```

Comment:
should these two be documented on which needs to be used at which case?

Reply:
these are just standard delegate methods not accessible by anyone

you may be familiar, but these type of `Delegate`'s (/patterns) are quite common in iOS and documenting them would be akin to documenting every function written in a class (or, in other words, not much value / its redundant)